templates/Web/_composition/Flask/Page.TheFlask.Grid.AddMerging/.template.config/template.json

```diff
@@ -0,0 +1,28 @@
+{
+  "author": "Microsoft Community",
+  "classifications": [
+    "Universal"
+  ],
+  "name": "Page.Flask.Grid.AddMerging",
```

Comment:
You need to change this to `Page.TheFlask.Grid.AddMerging`

Reply:
I was thinking that one too. Thanks for the catch! PR updated.